azurerm/resource_arm_key_vault_access_policy.go

```diff
@@ -100,6 +100,8 @@ func resourceArmKeyVaultAccessPolicy() *schema.Resource {
 			"key_permissions": azure.SchemaKeyVaultKeyPermissions(),
 
 			"secret_permissions": azure.SchemaKeyVaultSecretPermissions(),
+
+			"storage_permissions": azure.SchemaKeyVaultStoragePermissions(),
```

Comment:
could we [add/update a test to cover this field](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/resource_arm_key_vault_access_policy_test.go#L349-L353) add this new field [to the documentation](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/key_vault_access_policy.html.markdown#L89)?